modules/rostests/apitests/imm32/JapanImeConvTest.h

```diff
@@ -0,0 +1,259 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for Japanese IME conversion
+ * COPYRIGHT:   Copyright 2022 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include <ntstatus.h>
```

Comment:
For which thing is this needed? You don't seem to use any NTSTATUS values (at least explicitly) there.

Reply:
Deleted.